modules/core/src/main/scala/org/scalasteward/core/forge/gitlab/GitLabApiAlg.scala

```diff
@@ -270,28 +298,86 @@ final class GitLabApiAlg[F[_]: Parallel](
       case mr =>
         logger.info(s"Unable to automatically merge ${mr.webUrl}").map(_ => mr)
     }
+  import cats.implicits._
 
-  private def maybeSetReviewers(repo: Repo, mrOut: MergeRequestOut): F[MergeRequestOut] =
-    gitLabCfg.requiredReviewers match {
-      case Some(requiredReviewers) =>
-        for {
-          _ <- logger.info(
-            s"Setting number of required reviewers on ${mrOut.webUrl} to $requiredReviewers"
+  private def setReviewers(
```

Comment:
It might be worth adding a comment here reminding readers that this is retained only for Gitlab versions older than... 12.3 I think?